sdk/eventgrid/Azure.Messaging.EventGrid/src/autorest.md

```diff
@@ -3,6 +3,49 @@
 Run `dotnet msbuild /t:GenerateCode` to generate code.
 
 ``` yaml
+directive:
+- from: swagger-document
+  where: $.definitions
+  transform: >
+    for (var path in $)
+    {
+      if (!path.includes("CloudEvent") && !path.includes("EventGridEvent"))
+      {
+        $[path]["x-namespace"] = "Azure.Messaging.EventGrid.Models.SystemEvents";
```

Comment:
nit: I think you can pull out the namespace into a constant:
```js
const namespace = "Azure.Messaging.EventGrid.SystemEvents";
```
Also, drop the `Models`.